class-debug-bar-cron.php

```diff
@@ -380,6 +382,21 @@ private function display_schedules() {
 				</table>';
 		}
 
+		/**
+		 * Sorting method for cron scheldules. Order by schedules interval.
+		 *
+		 * @param array $a First element of comparison pair.
+		 * @param array $b Second element of comparison pair.
+		 *
+		 * @return int Return 1 if $a argument 'interval' greater then $b argument 'interval', 0 if both intervals equivalent and -1 otherwise.
+		 */
+		function schedules_sorting( $a, $b ){
+			if ( (int) $a['interval'] === (int) $b['interval'] ){
```

Comment:
WP Coding standards demands a space between these `){` => `) {`. Line 393 and 394.